src/features/app_start/shared/app_start_info_button/index.tsx

```
@@ -0,0 +1,86 @@
+            border: '1px solid var(--accent-300)',
+            gap: '8px',
+            padding: '16px',
+            backgroundColor: 'var(--accent-100)',
+            marginBottom: '16px',
+            display: 'flex',
+            flexDirection: 'row',
+          }}
+        >
+          <Typography
+            className="h4"
+            dangerouslySetInnerHTML={{ __html: bannerMessage }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)